js/scripts/release.sh

```diff
@@ -68,11 +68,12 @@ if [ "$BRANCH" == "master" ]; then
 fi
 
 echo "*** Updating cargo parity-ui-precompiled#$PRECOMPILED_HASH"
+git submodule update
 cargo update -p parity-ui-precompiled
 # --precise "$PRECOMPILED_HASH"
 
 echo "*** Committing updated files"
-git add .
+git add Cargo.lock
```

Comment:
Also needs `js/package.json` if new version was created. ("Bumping package.json above conditionally updates - if updated, needs to be added as well)